server/chat-plugins/room-events.ts

```diff
@@ -522,6 +522,7 @@ export const commands: ChatCommands = {
 			}
 
 			// rebuild the room.settings.events object
+			room.settings.events = {};
```

Comment:
This looks like it would clear all aliases and categories out of the event table, which would be bad.

Reply:
You're right, I forgot that table contained more than just the raw events. 
Would you recommend adding `delete room.settings.events[eventId]` on the line before `room.settings.events[eventId] = sortedObj;` instead? That'd work but I don't know enough about optimization to know if that's good enough

Reply:
Yeah, I think that's a reasonably good solution.